### PR TITLE
Improve the clarity of the installation guide

### PR DESCRIPTION
**Pull request by zsnmwy**

### CODE REVIEW

File: charts/clusternet-agent/README.md [hunk 1]
```diff
@@ -13,12 +13,27 @@ helm install clusternet-agent -n clusternet-system --create-namespace \
...
   --set parentURL=https://192.168.10.10:6443 \
   --set registrationToken=07401b.f395accd246ae52d \
   clusternet/clusternet-agent
 ```
 
+**Note:** If bootstrapping authentication is **not supported** by the kube-apiserver in parent cluster (like k3s) , i.e. `--enable-bootstrap-token-auth=false` (which defaults to be false), please use serviceaccount token instead.
+
+```console
+# Get serviceaccount token from parent cluster.
+$ kubectl get secret -n clusternet-system -o=jsonpath='{.items[?(@.metadata.annotations.kubernetes\.io/service-account\.name=="cluster-bootstrap-use")].data.token}' | base64 --decode; echo
```
Comment (dixudx) [inline charts/clusternet-agent/README.md:26]: Well, this command is not run in child cluster. Better not add this here, which may mislead users.

Comment (dixudx) [reply charts/clusternet-agent/README.md:26]: Could we add a link on how to get or generate this token?

**Review by dixudx — COMMENTED** [review]
Nice work. Thanks for your contribution.

File: charts/clusternet-agent/README.md [hunk 2]
```diff
@@ -13,12 +13,23 @@ helm install clusternet-agent -n clusternet-system --create-namespace \
...
-helm install clusternet-agent -n clusternet-system --create-namespace \
+$ helm install clusternet-agent -n clusternet-system --create-namespace \
   --set parentURL=https://192.168.10.10:6443 \
   --set registrationToken=07401b.f395accd246ae52d \
   clusternet/clusternet-agent
 ```
 
+**Note:** If bootstrapping authentication is **not supported** by the kube-apiserver in parent cluster (like k3s) , i.e. `--enable-bootstrap-token-auth=false` (which defaults to be false), please use serviceaccount token instead.
+
+[Click here to get the serviceaccount token from parent cluster.](../clusternet-hub/README.md#get-serviceaccount-token-from-parent-cluster)
```
Comment (zsnmwy) [inline charts/clusternet-agent/README.md:24]: Should be clearer than before

**Comment by dixudx** [comment]
```
------------------------------------------------------------------------------------------------------------------------
 ✖︎ clusternet-agent => (version: "0.3.0", path: "charts/clusternet-agent") > Chart version not ok. Needs a version bump!
 ✖︎ clusternet-hub => (version: "0.3.0", path: "charts/clusternet-hub") > Chart version not ok. Needs a version bump!
------------------------------------------------------------------------------------------------------------------------
```

@zsnmwy Please bump a newer version instead. Thanks.

**Pull request merged**